README.md

```diff
@@ -361,21 +361,14 @@ driver:
   name: ec2
   aws_ssh_key_id: id_rsa-aws
   security_group_ids: ["sg-1a2b3c4d"]
-  region: us-east-1
-  availability_zone: b
+  region: us-west-2
+  availability_zone: us-west-2b
```

Comment:
I know this was broken earlier, but I would like to encourage people to use the single-letter form of this config to DRY this yaml up a bit.  Would you try `b` with the latest version of kitchen-ec2 (0.9.5+)?